registrations/management/commands/pseudonymize_past_signups.py

```diff
@@ -0,0 +1,43 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.utils.timezone import localtime
+
+from registrations.models import SignUp, SignUpGroup
+
+
+class Command(BaseCommand):
+    help = (
+        "Pseudonymize signups and signup groups for past events. "
+        "The time of pseudonymization can be specified in days in the "
+        "PSEUDONYMIZATION_THRESHOLD_DAYS environment variable."
+    )
+
+    def handle(self, *args, **options):
+        threshold_time = localtime() - timedelta(
+            days=settings.PSEUDONYMIZATION_THRESHOLD_DAYS
+        )
+        signup_groups = SignUpGroup.objects.filter(
```

Comment:
```suggestion
        signup_groups = SignUpGroup.objects.prefetch_related("signups").filter(
```
Since a group's pseudonymization will use `signups.all()`

Reply:
Added the prefetch for signups